ixmp4/data/db/iamc/timeseries/repository.py

```diff
@@ -127,9 +127,16 @@ def map_measurand(df):
             df["unit__id"] = unit__id
             return df
 
+        # ensure compatibility with pandas < 2.2
+        # TODO remove legacy-handing when dropping support for pandas < 2.2
```

Comment:
```suggestion
        # TODO remove legacy-handling when dropping support for pandas < 2.2
```
Or did you mean `handing`?

Reply:
Thanks for catching that, fixed it.